README.md

```diff
@@ -18,10 +18,14 @@ helping out. Simply fill out the [Climatescape Contributor Application][contribu
     $ npm install
     ```
 2. Copy your Airtable API key from this page: [Airtable account][airtable-account]
-3. Run the project like so, replacing `YOUR_KEY_HERE` the key you copied in the
-    previous step:
+3. Create a `.env.development` and a `.env.production` file at the root of the project
+   replacing `YOUR_KEY_HERE` by the key you copied in the previous step:
     ```bash
-    $ AIRTABLE_API_KEY=YOUR_KEY_HERE npm run develop
+    $ echo "AIRTABLE_API_KEY=YOUR_KEY_HERE" > .env.development && echo "AIRTABLE_API_KEY=YOUR_KEY_HERE" > .env.production
+    ```
```

Comment:
I've seen some projects have a `.env.development.sample` or use a placeholder.

Also, do users getting started need to think about a `.env.production`? Perhaps, it would make sense to have a separate section for deploying your own version with an airtable fork.

Reply:
@jasonLaster you're right about the production file–I'll make that small change after merging. As far as the sample, I think it would be a great contribution if someone wants to submit a PR for that 🚀